src/hostedPages/activities/form/TraditionalForm.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { Button, CircularSpinner } from '../../../atoms'
+import classes from './form.module.scss'
+import { Question } from '../../../molecules'
+import { useWizardForm } from '../../../hooks/useWizardForm'
+import layoutClasses from '../../layouts/HostedPageLayout/hostedPageLayout.module.scss'
+import { FormProps } from '../../../types/form'
+
+export const TraditionalForm = ({
+  form,
+  onSubmit,
+  buttonLabels,
+  evaluateDisplayConditions,
+  errorLabels,
+  questionTypeConfig = {},
+  storedAnswers,
+  onAnswersChange,
+}: FormProps) => {
+  const {
+    submitForm,
+    formMethods: { control, getValues },
+    errors,
+    isEvaluatingQuestionVisibility,
+  } = useWizardForm({
+    questions: form.questions,
+    onSubmit,
+    evaluateDisplayConditions,
+    errorLabels,
+    storedAnswers,
+    onAnswersChange,
+  })
```

Comment:
Have you tested the TraditionalForm component with Display Logic already or is that something you can only do in Awell Hosted Pages? Display conditions have to be evaluated every time an answer changes (including some form of debounce). This means the array of `visibleQuestions` changes every time you complete a question. Based on the code here, it's unclear to me whether display logic will work.

Also, it might be confusing that the `TraditionalForm` component also uses the `useWizardForm` hook? Maybe it deserves its own hook? Or is there enough overlap and no negative consequences to re-use it?

Also, see https://github.com/awell-health/orchestration-stories/blob/main/pages/stories/form/index.tsx for reference.